opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -457,6 +457,7 @@ class Well {
     bool updateGroup(const std::string& group);
     bool updateWellGuideRate(bool available, double guide_rate, GuideRateTarget guide_phase, double scale_factor);
     bool updateWellGuideRate(double guide_rate);
+    bool updateWellGuideRate(bool available);
```

Comment:
this function has pretty specific purpose to set whether the well is available from group control, maybe the function name can be `updateAvailabilityFromGroupControl` for better readability.

Reply:
Changed it to `updateAvailableForGroupControl`